material-popup-menu/src/main/java/com/github/zawadz88/materialpopupmenu/ViewBoundCallback.kt

```diff
@@ -0,0 +1,31 @@
+package com.github.zawadz88.materialpopupmenu
+
+import android.view.View
+
+/**
+ * Callback to be invoked once the custom item view gets created and bound.
+ * It is to be used when some views inside need to be updated once inflated.
+ *
+ * This class is an extension for closure based callback that provides
+ * additional functionality such as dismissing popup.
+ *
+ * @param callback block of the callback in which you can bind the given view
+ */
+class ViewBoundCallback(
+    private val callback: ViewBoundCallback.(View) -> Unit
+) : (View) -> Unit {
```

Comment:
Ha quite clever to extent a closure here ;)

Reply:
Yeah that's a nice feature. I found it by accident :)